Releases.md

```diff
@@ -0,0 +1,84 @@
+* [Introduction](#introduction)
+* [Versioning](#versioning)
+* [Tagging repositories](#tagging-repositories)
+* [Components](#components)
+* [Release checklist](#release-checklist)
+* [Release process](#release-process)
+
+## Introduction
+
+This document provides details about Kata Containers releases.
+
+## Versioning
+
+The Kata Containers project uses [semantic versioning](http://semver.org/) for all releases. Semantic versions are comprised of three fields in the form:
+
+```
+MAJOR.MINOR.PATCH
+```
+
+For examples: `1.0.0`, `1.0.0-rc.5`, and `99.123.77+foo.bar.baz.5`.
+
+Semantic versioning is used since the version number is able to convey clear information about how a new version relates to the previous version. It can also provide assurances to allow users to know for example when they must upgrade compared with when they may wish to upgrade:
+
+- When `PATCH` increases, the new release contains important **security fixes**
+  and an upgrade is recommended.
+
+  The patch field can contain extra details after the number. Dashes denote pre-release versions. `1.0.0-rc.5` in the example denotes the fifth release candidate for release `1.0.0`. Plus signs denote other details. In our example, `+foo.bar.baz.5` is additional information regarding release `99.123.77` in the example above.
+
+- When `MINOR` increases, the new release adds **new features** but *without
+  changing the existing behavior*.
+
+- When `MAJOR` increases, the new release adds **new features, bug fixes, or
+  both** and which *changes the behavior from the previous release* (incompatible with previous releases).
+
+  A major release will also likely require a change of the container manager version used, for example Docker\*. Please refer to the release notes for further details.
+
+## Tagging repositories
+
+To create a signed and annotated tag for a repository, first ensure that `git(1)` is configured to use your `gpg(1)` key:
+
+```
+$ git config --global user.signingkey $gpg_key_id
+```
+
+To create a signed and annotated tag:
+
+```
+$ git tag -as $tag
+```
+
+The tag name (`$tag` in the example above) must conform to the [versioning](#versioning) an example being `1.0.0-rc2`.
```

Comment:
The tag name (`$tag` in the previous example) must conform to the [versioning](#versioning) (e.g. `1.0.0-rc2`).